### DISCUSSION TITLE
2024/12/16 Co-working Meeting

### DISCUSSION THREAD

**Discussion by astroDimitrios** [carpentries-incubator/intro-to-modern-fortran#32] [Meetings]
There needs to be a standard for what to call example programs used in the lesson and challenge programs.

We spoke about Episodes 1 and 2 and agreed Episode 2 should stop around where @theabro is up to atm in #29 and that we can split up the rest of the 'Basic' content into maybe two more episodes.

@TomMelt and Andrew Brown worked on getting some content in for #33

**Comment by theabro** [comment]
Discussed PR #29 - move `implicit none` to Ep1, make examples more of one big program. Rename to be example201 etc.
